revolv/project/models.py

```diff
@@ -537,4 +550,33 @@ class DonationLevel(models.Model):
     """
     project = models.ForeignKey(Project)
     description = models.TextField()
-    amount = models.IntegerField()
+    amount = models.DecimalField(
+        max_digits=15,
+        decimal_places=2
+    )
+
+
+class MonthlyReinvestableAmount(models.Model):
+    """
+    Part of the money to be repaid to RE-volv per month for a particular project
+    that can be made available for reinvestment.
```

Comment:
Here would be a good place to explain how the monthly reinvestable amounts work with respect to the project (e.g. that there exists one monthly reinvestable amount for each year and to calculate what amount should be reinvested, we look at the one for the current year)